command/issue.go

```diff
@@ -85,7 +85,8 @@ func issueList(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	baseRepo, err := determineBaseRepo(cmd, ctx)
+	repo, _ := cmd.Flags().GetString("repo")
+	baseRepo, err := determineBaseRepo(cmd, ctx, repo)
```

Comment:
Since we're already passing `cmd` to `determineBaseRepo`, could we look up the "repo" flag inside `determineBaseRepo` instead of having to modify all the callers like this?

I think it would help apply this fix with a much smaller diff.

Reply:
sure. thanks, will do.

Reply:
@mislav done